### ISSUE TITLE
Crash in BugsnagClient.m line 818

### ISSUE THREAD

**Issue by sethfri** [bugsnag/bugsnag-cocoa#900]
### Describe the bug
Crash of unknown origin coming from Bugsnag. We have seen 1 crash coming from 1 user so far.

### Steps to reproduce
Unknown

### Environment
* Bugsnag version: 6.2.3
* CocoaPods version: 1.9.3
* Carthage version (if any): n/a
* iOS/tvOS/macOS version(s): 14.0.1
* Simulator or physical device: iPhone 8 Plus
* Xcode version: 12.1
* Swift version (if applicable): 5.1

### Stack Trace

<details><summary>Stack trace:</summary>

```
Fatal Exception: NSInvalidArgumentException
-[NSOperationQueue addAutoBreadcrumbOfType:withMessage:andMetadata:]: unrecognized selector sent to instance 0x107e18e50

0  CoreFoundation                 0x198d31114 __exceptionPreprocess
1  libobjc.A.dylib                0x1ac557cb4 objc_exception_throw
2  CoreFoundation                 0x198c419b8 -[NSOrderedSet initWithSet:copyItems:]
3  CoreFoundation                 0x198d33758 ___forwarding___
4  CoreFoundation                 0x198d356cc _CF_forwarding_prep_0
5  MyApp                           0x1054df0d4 __42-[BugsnagClient setupConnectivityListener]_block_invoke + 818 (BugsnagClient.m:818)
6  MyApp                           0x1054d6604 BSGConnectivityCallback + 97 (BSGConnectivity.m:97)
7  SystemConfiguration            0x1991cce44 reachPerformAndUnlock
8  SystemConfiguration            0x1991cd6e0 ____SCNetworkReachabilityRestartResolver_block_invoke_2
9  libnetwork.dylib               0x199918930 __nw_resolver_update_client_block_invoke
10 libdispatch.dylib              0x198969298 _dispatch_call_block_and_release
11 libdispatch.dylib              0x19896a280 _dispatch_client_callout
12 libdispatch.dylib              0x1989464fc _dispatch_lane_serial_drain$VARIANT$armv81
13 libdispatch.dylib              0x198946fe8 _dispatch_lane_invoke$VARIANT$armv81
14 libdispatch.dylib              0x198950808 _dispatch_workloop_worker_thread
15 libsystem_pthread.dylib        0x1de6145a4 _pthread_wqthread
16 libsystem_pthread.dylib        0x1de617874 start_wqthread
```
</details>

**Comment by johnkiely1** [comment]
Hi @sethfri ,

Thanks for raising, we will investigate and get back to you.

**Comment by mattdyoung** [comment]
This is now fixed in [v6.2.6](https://github.com/bugsnag/bugsnag-cocoa/releases/tag/v6.2.6).
